pkgs/stdenv/linux/default.nix

```diff
@@ -6,16 +6,26 @@
 { lib
 , localSystem, crossSystem, config, overlays
 
-, bootstrapFiles ? { # switch
-    "i686-linux" = import ./bootstrap-files/i686.nix;
-    "x86_64-linux" = import ./bootstrap-files/x86_64.nix;
-    "armv5tel-linux" = import ./bootstrap-files/armv5tel.nix;
-    "armv6l-linux" = import ./bootstrap-files/armv6l.nix;
-    "armv7l-linux" = import ./bootstrap-files/armv7l.nix;
-    "aarch64-linux" = import ./bootstrap-files/aarch64.nix;
-    "mips64el-linux" = import ./bootstrap-files/loongson2f.nix;
-  }.${localSystem.system}
-    or (abort "unsupported platform for the pure Linux stdenv")
+, bootstrapFiles ?
+  let table = {
```

Comment:
Maybe we should just use config?

Reply:
I don't think "config" is  canonical for a a given system/libc pair.  I'm not sure how important that is, but that's why I did things this way.

Reply:
Yeah makes sense.